cosmo_tester/test_suites/cli/__init__.py

```diff
@@ -57,6 +63,85 @@ def _test_cfy_install(run, example, paths, logger):
     example.check_files()
 
 
+def _set_ssh_in_profile(run, example, paths):
+    run(
+        '{cfy} profiles set --ssh-user {ssh_user} --ssh-key {ssh_key}'.format(
+            cfy=paths['cfy'],
+            ssh_user=example.manager.username,
+            ssh_key=paths['ssh_key'],
+        )
+    )
+
+
+def _test_cfy_logs(run, cli_host, example, paths, logger):
+    _set_ssh_in_profile(run, example, paths)
+
+    linebreak = '\\r\\n' if 'C:\\' in paths['cfy'] else '\n'
+    logger.info('Testing `cfy logs download`')
+    logs_dump_filepath = str(run(
+        '{cfy} logs download '.format(cfy=paths['cfy'])
+    ).stdout).split('archive to:')[1].split(linebreak)[0]
```

Comment:
Does this command not provide json output?

Reply:
(because we don't want to be doing horrible stuff like splitting on linebreaks- which I think .splitlines would do if we absolutely couldn't avoid it)

Reply:
So to answer that question: Yes, it has --json output... which outputs nothing. Fantastic.